pkg/legacy/importer_test.go

```diff
@@ -42,16 +42,15 @@ func TestGetAgentConfig(t *testing.T) {
 
 		valueStr := python.PyString_AS_STRING(value.Str())
 
+		goValue, found := agentConfigGo[keyStr]
 		// histogram_percentiles were converted from string to float
 		// by the config module in agent5. In agent6 this is now the
 		// responsibility of the histogram class.
 		// The value is overwritten anyway: we're just testing the
 		// default value.
 		if keyStr == "histogram_percentiles" {
-			valueStr = "['0.95']"
+			goValue = "[0.95]"
```

Comment:
Should this not be a list of string ?

Reply:
The agent5 conf recognizes a single input as `float` instead of a string. But given a comma-separated list, the config will be recognized as a string and converted to list of strings.